sdk/monitor/azure-monitor-opentelemetry-exporter/src/main/java/com/azure/monitor/opentelemetry/exporter/implementation/models/Response.java

```diff
@@ -0,0 +1,106 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
```

Comment:
was this generated by AutoRest?

Reply:
no, this class wasn't there initially. autorest only converts existing data models.

Reply:
it's missing this "// Code generated by Microsoft (R) AutoRest Code Generator." since it's not auto-generated, i will leave it like this.

Reply:
> i can rerun but it will mess up telemetryitem class

can you rerun and then just undo checkout on the telemetryitem class?